node.js/index.js

```diff
@@ -0,0 +1,114 @@
+// External Libraries:
+const agent = require('agentkeepalive');
+const request = require('request');
+
+// Internal Libraries:
+const constants = require('./constants');
+
+// Getting Environment Variables:
+const getConfig = () => {
+    var config = {};
+
+    if (process.env.LOGDNA_KEY) config.key = process.env.LOGDNA_KEY;
+    if (process.env.LOGDNA_HOSTNAME) config.hostname = process.env.LOGDNA_HOSTNAME;
+
+    if (process.env.LOGDNA_TAGS && process.env.LOGDNA_TAGS.length > 0) {
+        config.tags = process.env.LOGDNA_TAGS.split(',').map(tag => tag.trim()).join(',');
+    }
+
+    return config;
+};
+
+// Message Sanity Check:
```

Comment:
This is a style thing but I'd get rid of the colons on all your comments; it reads odd to me.

Reply:
Done!